### PR TITLE
[BBC-9064] Storybook: PhoneField.mdx

### PR DESCRIPTION
**Pull request by gabrielagabriel**
## Description

screencapture-localhost-4000-2020-11-02-19_19_59

## What has been done

- To MDX

## Things to consider

- A11y is broken... @nickoola  
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md

## How it was tested

- UT

### CODE REVIEW

**Review by mcellier — APPROVED** [review]

File: src/phoneField/PhoneField.story.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    <PhoneField id="phoneField" name="phoneField" onChange={() => {}} />
+  </Story>
+</Preview>
+
+### Custom
+
+<Preview>
+  <Story name="Custom">
+    <React.Fragment>
+      <label id="phoneFieldId" htmlFor="phoneFieldId">
```
Comment (nickoola) [inline src/phoneField/PhoneField.story.mdx:23]: Remove the `htmlFor="phoneFieldId"` on the label tag. 
Remove `id="phoneFieldId"` as already on the `label` (need to be unique).
Update `aria-labelledby="labelId"` to `aria-labelledby="phoneFieldId"` on the `PhoneField ` component.

File: src/phoneField/PhoneField.story.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+Maximum width of the name should be in proportion of 1:2 to the number.
+If the name is longer than available field it should get truncated.
+
+## Usage
+
+```jsx
+import { PhoneField } from '@blablacar/ui-library/build/phoneField'
+
+<label><PhoneField name="phoneField" onChange={() => {}} /></label>
+<label htmlFor="ID">
```
Comment (nickoola) [inline src/phoneField/PhoneField.story.mdx:75]: The `<label htmlFor="ID">` should not wrap the `PhoneField` component and should follow my ather remark above.

File: src/phoneField/PhoneField.story.mdx (hunk 1 above)
Comment (nickoola) [reply src/phoneField/PhoneField.story.mdx]: To fix the linter error you could replace the <label> with a <heading> and it'll more correspond to the way we use the `PhoneField` component.

**Review by wakooka — APPROVED** [review]

File: src/phoneField/PhoneField.story.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+Maximum width of the name should be in proportion of 1:2 to the number.
+If the name is longer than available field it should get truncated.
+
+## Usage
+
+```jsx
+import { PhoneField } from '@blablacar/ui-library/build/phoneField'
+
+  <label id="labelId">Label</label>
```
Comment (nickoola) [inline src/phoneField/PhoneField.story.mdx:75]: I think eslint will return an error on this one too. `<label>` >> `<p>` should fix.

**Pull request merged**